projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -277,21 +272,15 @@ export class TableWidgetRendererComponent
   }
 
   public onSelectChange(changed: SelectChange): void {
-    switch (changed.value.type) {
-      case TableControlOptionType.UnsetFilter:
-        this.selectFilterSubject.next(
-          this.selectFilterSubject.getValue().filter(existingFilter => existingFilter.field !== changed.value.metaValue)
-        );
-        break;
-      case TableControlOptionType.Filter:
-        this.selectFilterSubject.next(this.mergeFilters(changed.value.metaValue));
-        break;
-      case TableControlOptionType.Property:
-        this.queryPropertiesSubject.next(this.mergeQueryProperties(changed.value.metaValue));
-        break;
-      default:
-        assertUnreachable(changed.value);
+    if (changed.values.length === 0) {
+      // tslint:disable-next-line: no-void-expression
+      return this.selectFilterSubject.next(this.removeFilters(changed.select.options[0].metaValue.field));
```

Comment:
we don't need the return here right?

Reply:
Don't _need_ to, but returning makes the code a little cleaner because we don't need an `else` block or a separate  `return;` statement within the `if` block. I could eliminate the `return` below, I suppose with the same results. It just looked odd.

Reply:
I honestly find this more confusing. If we can use a separate `return` i think that's more clear.

Reply:
This is the strangest thing to hold up a PR for.

Reply:
I'll change it in my local.

Reply:
It is a strong nit from my end

Reply:
I'm going to change it. Still silly to hold up a PR over this.